legend-sdlc-server/src/main/resources/docker/config/config.json

```diff
@@ -0,0 +1,106 @@
+{
+  "applicationName": "Legend SDLC",
+  "server": {
+    "adminContextPath": "/admin",
+    "applicationConnectors": [
+      {
+        "type": "http",
+        "port": ${SDLC_PORT},
+        "maxRequestHeaderSize": "128KiB"
+      }
+    ],
+    "adminConnectors": [
+      {
+        "type": "http",
+        "port": 7071
+      }
+    ],
+    "gzip": {
+      "includedMethods": [
+        "GET",
+        "POST"
+      ]
+    },
+    "requestLog": {
+      "type": "classic",
+      "level": "OFF",
+      "appenders": [
+        {
+          "type": "console",
+          "logFormat": "OFF"
+        }
+      ]
+    },
+    "rootPath": "/api"
+  },
+  "projectStructure": {
+    "projectCreation": {
+      "groupIdPattern": "^org\\.finos\\.legend\\..+"
+    },
```

Comment:
Who are the target users of this config? Is org.finos.legend an appropriate prefix for group ids for these users?

Reply:
The target users are newcomers to legend who want to run the Legend stack to get a feel for what it is/does.

I understand your question about the groupIdPattern. Maybe I can change it to "org.demo.legend" ? 

In any case, the target user does not know enough of Legend to even reason about this config effectively.

Reply:
I don't think a groupId pattern is required at all for this kind of user. I would recommend simply deleting it.